library/packages/src/modules/PackageSystem.rb

```diff
@@ -33,6 +33,7 @@
 # The documentation is maintained at
 # <a href="../index.html">.../docs/index.html</a>.
 require "yast"
+require "yast2/execute"
 require "shellwords"
```

Comment:
should no longer be needed if it is used only in that one instance for shellescape

Reply:
Well, it is used also in `PackageInstalled` method. So we could consider adapt it to use Yast::Execute in case we want to completely remove the 'shellwords' requirement